data/35e/wizards_of_the_coast/campaign_settings/forgotten_realms/races_of_faerun/rac_abilities.lst

```diff
@@ -64,6 +64,15 @@ SOURCELONG:Races of Faerun	SOURCESHORT:Rac	SOURCEDATE:2004-03
 #- Favored Class: Barbarian.
 
 
+Mountain Orc Racial Traits
```

Comment:
Multiline is not a default setting in pcgen.  For that reason, I only accepted single line submissions.